react-challenge.html

```diff
@@ -35,30 +47,35 @@
 
         handleMoodOptions = () => {
           let randomIndex = Math.floor(Math.random() * this.state.moods.length )
-          this.setState({ currentMood: moods[randomIndex] })
+          this.setState({this.state.currentMood: this.state.moods[this.state.randomIndex] })
         }
 
+
+
         render(){
           return(
 
               <h1>Welcome to the Mood Meter App</h1>
 
               <h3>Mood Options:</h3>
 
-              <div style={{ fontSize: "40px", height: "50px"}}>
-                { this.state.currentMood }
+              <div style={{ this.fontSize: "40px", height: "50px"}}>
+                {{ this.state.currentMood }}
```

Comment:
```suggestion
                { this.state.currentMood }
```
Only one set of curly braces is needed to execute Javascript. Two sets of curly braces are needed for `style={{}}` but that's a special case